.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Question
+labels: question
+about: Ask a general question related to OpenSSL development
+---
+
+<!--
+Thank you for your interest in OpenSSL. If this is your first question,
+please take the time to read the following lines before posting it.
+
+NOTE:
+
+    If you're asking about how to use OpenSSL, this isn't the right 
+    forum.  Please see our User Support resources:
+    https://github.com/openssl/openssl/blob/master/.github/SUPPORT.md
+
+For questions related to build issues:
+
+    Please use the 'Bug report' template.
+
+For other questions:
+
+    Please describe your problem as concise as possible while giving
+    us enough information to understand your problem. An example code
```

Comment:
Drop the _An_.  I.e. _Example code or commands are_

Reply:
Addressed in 0c4dbb5cdc49f1213d66775c8ebe9e0aeba67da8.